recipes/pyftpdlib/meta.yaml

```diff
@@ -0,0 +1,55 @@
+{% set name = "pyftpdlib" %}
+{% set version = "1.5.0" %}
+{% set sha256 = "88c1dc3daa8de9ff5b25fd57ae8cc4390b5d034f87f2396ebd93eeb8f6879972" %}
+
+package:
+  name: {{ name|lower }}
+  version: {{ version }}
+
+source:
+  fn: release-{{ version }}.tar.gz
+  url: https://github.com/giampaolo/pyftpdlib/archive/release-{{ version }}.tar.gz
+  sha256: {{ sha256 }}
+
+build:
+  number: 0
+  script: python setup.py install --single-version-externally-managed --record record.txt
+  entry_points:
+    - pyftpdlib = clcache:main
+
+requirements:
+  build:
+    - python
+    - setuptools
+  run:
+    - python
```

Comment:
We do not have a clear policy on "extra" (optional) dependencies. I usually add those that I believe are important. Since I am not a user of `pyftpdlib` I cannot say anything about `PyOpenSSL`. Do you have any strong reason to request that @synapticarbors?